utilities/src/main/java/io/syndesis/qe/TestConfiguration.java

```diff
@@ -51,6 +51,10 @@
     public static final String SYNDESIS_CRD_URL = "syndesis.config.crd.url";
     public static final String SYNDESIS_OPERATOR_IMAGE = "syndesis.config.operator.image";
     public static final String SYNDESIS_CR_URL = "syndesis.config.cr.url";
+    public static final String SYNDESIS_DEFAULT_CR_VERSION = "syndesis.config.cr.version";
+    //if CR from the resources should use instead of upstream CR
+    public static final String SYNDESIS_CR_FROM_RESOURCES = "syndesis.config.cr.from.resources";
```

Comment:
this is not needed, when you want to use different CR, you can use `syndesis.config.cr.url`

Reply:
I see, I didn't realize that for using the CR from the resources or for using the different version of the `default-cr.yaml`, only URL `syndesis.config.cr.url` can be used.

e.g.
```
# file from the resources
syndesis.config.cr.url=https://raw.githubusercontent.com/syndesisio/syndesis-qe/master/utilities/src/main/resources/syndesis-minimal.yaml
# different version
syndesis.config.cr.url=https://raw.githubusercontent.com/syndesisio/fuse-online-install/1.9.x/default-cr.yml
```
is it correct? If yes, i will remove the whole section https://github.com/syndesisio/syndesis-qe/pull/1351/files#diff-a647f00004fb801decc88f91795a5ca9R383-R394 and I add there only 
```
if (properties.getProperty(SYNDESIS_CR_URL) == null) {
   defaultProps.setProperty(SYNDESIS_CR_URL, "https://raw.githubusercontent.com/syndesisio/fuse-online-install/master/default-cr.yml");
}
```

Reply:
yes, should be ok